cmd/util.go

```diff
@@ -148,3 +148,23 @@ func checkIfRemotePath(p string) bool {
 func parseScpURL(p string) (string, string) {
 	return strings.Split(p, ":")[0], strings.Split(p, ":")[1]
 }
+
+// pushSSHKey executes PushSSKey on EC2 resource types
+func pushSSHKey(r resources.Resource) error {
+	ec2, ok := r.(*resources.Ec2Instance)
+	if !ok {
+		return fmt.Errorf("pushing SSH key is supported only for EC2 instances")
+	}
+	color.PrintGreen(fmt.Sprintf("Sending SSH key to AWS SSM for %s (%s) in %s\n",
+		r.Name(), r.GetTag("environment"), r.GetZone()))
+	return ec2.PushSSHKey(config.GlobalConfig.EC2ConnectKeyPath)
+}
+
+// sshPrivKeyPath returns SSH private key path for a given public key
+func sshPrivKeyPath(publicKeyPath string) string {
```

Comment:
Shouldn't this be configurable as well? We can also try to auto-generate public key from private.

Reply:
It is auto-generated.